File: Lumina/benchmarks/fib.cpp
#include <iostream>

long long fib(long long n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    long long result = fib(35);
    std::cout << "Fib(35) = " << result << std::endl;
    return 0;
}

File: Lumina/benchmarks/fib.rs
fn fib(n: i64) -> i64 {
    if n <= 1 { return n; }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let result = fib(35);
    println!("Fib(35) = {}", result);
}
